docs/src/pages/components/data-grid/export/export.md

```diff
@@ -14,6 +14,16 @@ To enable the CSV export you need to compose a toolbar containing the `GridToolb
 
 {{"demo": "pages/components/data-grid/export/ExportSelectorGrid.js", "bg": "inline"}}
 
+### Only export some columns
+
+By default, the CSV will only contain the visible columns of the table.
+
+To export all columns, including the hidden one, you can use the `allColumns` option in the `csvOptions` property of `GridToolbarExport`
+
+To exclude additional columns, you can use the `disableExport` property available in `GridColDef`
+
+To define the exact columns you want to export, you can use the `fields` option in the `csvOptions` property of `GridToolbarExport`
```

Comment:
I think is missing which approach takes precedence. We have two main ways: `fields` and `disableExport`. If `fields` is false, then developers can also use `allColumns`.

```suggestion
To include or hide other columns, there are two ways:

1. Define the exact columns to be exported with the `fields` attribute in the `csvOptions` prop of [`GridToolbarExport`](/components/data-grid/components/#toolbar). 
Set `allColumns` in `csvOptions` to true to include hidden columns, instead of only the visible ones.

2. Set the `disableExport` attribute to true in each `GridColDef`.

<!-- demo -->
```